skbuild/resources/cmake/FindF2py.cmake

```diff
@@ -0,0 +1,98 @@
+#.rst:
+#
+# This module defines CMake functions to build Python extension modules and
+# stand-alone executables.
```

Comment:
Look  like the description should be inspired from text found at http://www.f2py.com/